tests/covscripts/coverage_parse_attributes.py

```diff
@@ -24,18 +24,19 @@
 #
 ###############################################################################
 import json
-import sys
+import os
 
-sys.path.append("../tuna")
-sys.path.append("tuna")
+root_dir = os.path.dirname(
```

Comment:
@cderb is there a better way to get the root  dir in Tuna without having to do this?

Reply:
Maybe by using relative path? "../../../"